apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -674,7 +674,7 @@ namespace MWScript
                             MWWorld::ConstContainerStoreIterator it = store.getSlot (slot);
                             if (it != store.end() && ::Misc::StringUtils::ciEqual(it->getCellRef().getRefId(), item))
                             {
-                                numNotEquipped -= it->getRefData().getCount();
+                                numNotEquipped -= std::abs(it->getRefData().getCount());
```

Comment:
A yet another public API change, that bloats PR size. Now count can be negative.

Probably it would be better to make `getCount()` to return positive value and add an another API to work with restockable items. Probably you can add an optional boolean flag to this method to allow client code to tell if it wants to work with restocking.

Reply:
I think I originally wanted to avoid changing getCount because non-store refs might already have negative counts. Not my brightest moment.